frame/nfts/src/benchmarking.rs

```diff
@@ -458,5 +458,28 @@ benchmarks_instance_pallet! {
 		}.into());
 	}
 
+	pay_tips {
+		let n in 0 .. T::MaxTips::get() as u32;
+		let amount = BalanceOf::<T, I>::from(100u32);
+		let caller: T::AccountId = whitelisted_caller();
+		let collection = T::Helper::collection(0);
+		let item = T::Helper::item(0);
+		let tips: BoundedVec<_, _> = vec![
+			ItemTip
+				{ collection, item, receiver: caller.clone(), amount }; T::MaxTips::get() as usize
+		].try_into().unwrap();
```

Comment:
```suggestion
		let tips: BoundedVec<_, _> = bounded_vec![
			ItemTip
				{ collection, item, receiver: caller.clone(), amount }; n as usize
		];
```

Reply:
It looks like we can't use bounded_vec! macro in benchmarks, probably because of `#[cfg(feature = "std")]`